docs/index.md

```diff
@@ -0,0 +1,617 @@
+
+![Pin layout TTGO T-Display V1.0](pic/TTGO_pin.jpg)
+
+To program the display and some voltages, example code can be found at [https://github.com/Xinyuan-LilyGO/TTGO-T-Display](https://github.com/Xinyuan-LilyGO/TTGO-T-Display)
+
+> 2020/04/07
+
+A second TTGO ESP32 arrived and is set to be programmed:
+
+![Two ESP32 TTGO](pic/ttgo.jpg)
+
+Without programming TTGO installed some software on the module. On the 135x240 display you get 22x30 characters with a 6x8 font. That's almost as much as my first ZX81 with 32x24. You can scan for nearby 2.4GHz networks, check the power supply and go into sleep mode. The last one is of interest for our project and the power consumption with these standard settings will be tested next.
+
+### Power consumption T-Display
+
+![TTGO-T-Display modes](pic/ttgo_disp1.jpg)
+
+> 2020/04/28
+
+With an external power supply we can check the power consumption of the TTGO. The values over USB were rather high, but it has a 3.7 LiPo battery connector and the power consumption there is significantly lower, most likely because the 5V to 3.3V step-down converter is not needed. Here are the values for 3.7 Volt over battery:
+
+- Power on, start up and screen on: 68 mA
+- WiFi scan for nearby hotspots: 108 mA
+- Sleep, waiting for interrupt from key pressed: 0.35 mA
+
+With a 1000 mAh battery and the given voltage we can calculate the power consumption and projected runtime:
+
+- Screen on: 252 mW, runtime 14 hours 42 minutes
+- WiFi on: 399 mW, runtime 9 hours 15 minutes
+- Hibernate: __1.3 mW__, runtime 2857 hours - or __119 days__
+
+> 2020/05/07
+
+School is back open since May 4th, students are back since May 5th - and now we got the solar panel and the battery! Time to find a place on the roof in Nha Be and control software to collect and transmit data.
+
+![Solar panel](pic/2020-05-07_solar.jpg)
+
+> 2020/05/08
+
+The wind generator arrived just one day later! Looked at location on top of the roof, 6th floor in Nha Be. Empty room for equipment is there, rain proved, and space for the 5 wired from solar and wind to the control unit. Maybe next week start first test setup?
+
+![Wind generator](pic/2020-05-08_wind.jpg)
+
+Data: __JLS-500__ with 24V and 500 W
+
+> 2020/05/14
+
+The TTGO can be programmed with MicroPython and [devbis](https://github.com/devbis) created both a [slow python driver](https://github.com/devbis/st7789py_mpy/) for the display ST7789 as well as a [fast C variant](https://github.com/devbis/st7789_mpy). Further description of this module [here](https://sites.google.com/site/jmaathuis/arduino/lilygo-ttgo-t-display-esp32). And the tft driver from Loboris is working as well, details in [this instuctable from February 2020](https://www.instructables.com/id/TTGO-color-Display-With-Micropython-TTGO-T-display/). Original data at [LilyGo github](https://github.com/Xinyuan-LilyGO/TTGO-T-Display).
```

Comment:
This actually writess about the TTGO T-Display. The company has other boards to offer!
![TTGO_ESP32](https://user-images.githubusercontent.com/43933271/99236740-276b1a00-282a-11eb-91b8-0f1a581fa3b3.jpg)

Reply:
Working on it. First need the folder for all the pictures.